htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -1721,6 +1721,11 @@ public function fetchAllPerMvt($piecenum, $mode = '')
 		dol_syslog(__METHOD__, LOG_DEBUG);
 		$result = $this->db->query($sql);
 		if ($result) {
+			// Add an empty line when transaction is validated to permit to add new line manually
```

Comment:
Hi @eldy

Try to add a manual transaction by clicking on the "+" button in general ledger. 
In this case fetchAllPerMvt will return an array with a "false" empty line. The line will be falsely empty because there will be a document number and journal code, but no account or amount. The software will therefore display an additional blank line.
So, if we add one more empty line in the array, the soft will display two lines with the "add" button, but only one can be stored.

By the way, I'm working on a PR to accept multi line in one action.

Cheers